src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
> So this NewRelic API doesn't have a notion of logging anything besides an error?

No other messages, as far as I can tell. It logs transactions (web, db, and others), but that's a different story.

> All you're be logging in this case is the exception and level. Not the namespace, logging message, args, etc.

The stack trace (and thus the namespace) is logged.

Sure we could log more stuff. TBH, I haven't even figured out how to get New Relic to show those "params" you pass to `noticeError`. It mostly cares about: exception class, exception message, and the containing transaction (a function which you can designate as such, usually).

There is also the notion of ["attributes"](https://docs.newrelic.com/docs/apm/applications-menu/events/viewing-apm-errors-error-traces). I might explore those in the future as a way to add more info to error logging, but for the time being I only wanted the basics.